### ISSUE TITLE
[Problem] Material system isn't accessible uniformly across the application

### ISSUE THREAD

**Issue by sliptonic** [FreeCAD/FreeCAD#10927]
The material system isn't a 'workbench'.  Rather it is a set of features and capabilities that are used within multiple workbenches.   Path, FEM, Arch and others will need to make use of it.  Therefore it makes sense for the capability to be launched and accessed without selecting a single workbench.

We need a button and or shortcuts globally available.

**Comment by luzpaz** [comment]
Typo in ticket title

**Comment by yorikvanhavre** [comment]
I see Material as a module, not a workbench. Like Tux, or MeshPart. It should basically furnish a series of FreeCAD commands, that are to be used in other workbenches, and maybe some sort of direct python functionality, if there is a need for it...

But while it's being built and tested, it might be convenient to have a workbench set up for quickly accessing/testing the different commands...

**Comment by sliptonic** [comment]
I still don't know why a workbench would be better at this stage than a module.  A module could still add a toolbar or panel.

**Comment by yorikvanhavre** [comment]
Yes, but so there is one centralized place to go look for these commands. But indeed it's a detail.

**Comment by luzpaz** [comment]
@sliptonic maybe we should close this in favor of #10174 and move the conversation over there ?
